src/components/sections/welcome.jsx

```diff
@@ -9,7 +9,9 @@ const Welcome = () => (
             <h1 className="header__title">HackBeanpot will be back!</h1>
             <h2 className="header__event-date">February 19-21, 2021</h2>
             <p className="header__description">Join us for our upcoming virtual hackathon!</p>
-            <NewsletterSignup />
+            <DynamicLink role="button" className="header__cta" to="https://apply.hackbeanpot.com/admin/settings">
```

Comment:
Could you change this link to just `https://apply.hackbeanpot.com` ? I think it should redirect to the login anyway if they're not logged in already, but it would still be good to fix just in case.

Reply:
whoops idk how i totally missed that